### DISCUSSION TITLE
Grand Prix Radio Syncing (liveLatency preset value + proces?)

### DISCUSSION THREAD

**Discussion by SmashRocket** [f1multiviewer/issue-tracker#105] [Feature requests]
Looking at most of the sessions the Grand Prix Radio 'liveLatency" value keeps being around 41,42,43 secs
Got a manual proces now of pauzing the audio, waiting for the buffer to build to a liveLatency of 42 secs en pressing play again.
Could this be something to automate?

- Press Grand Prix Radio button
- Pause stream 
- Build up buffer to X presetvalue (livelatency?)
- Press play again

Would like to change that presetvalue in the settings. 

(maybe also fixating that liveLatency to a static number so it isn't jumping around?)

**Comment by bartlangelaan** [comment]
Pressing L should move the audio to live, minus the liveLatency. This however not working currently, I opened a bug report for this: https://github.com/f1multiviewer/issue-tracker/discussions/117

If that is fixed, it is still not possible to change the default liveLatency for GPRadio, which I guess is the main frustration of manually syncing right now. So that's the only thing that needs to be added after the bug is fixed.
